store.go

```diff
@@ -0,0 +1,147 @@
+package cbornode
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	block "github.com/ipfs/go-block-format"
+	cid "github.com/ipfs/go-cid"
+	mh "github.com/multiformats/go-multihash"
+	recbor "github.com/polydawn/refmt/cbor"
+	atlas "github.com/polydawn/refmt/obj/atlas"
+	cbg "github.com/whyrusleeping/cbor-gen"
+)
+
+type IpldStore interface {
```

Comment:
Docs? Lots of docs...

Reply:
Added here: https://github.com/ipfs/go-ipld-cbor/pull/71